elastic/kibana
# [Security solution] Improve AI connector error handling

Improves error handling on Generative AI connectors.

To test: 

1. Save [this file](https://p.elstc.co/paste/ko7XA1Mk#uNrm4wgt-AWFIaR9iTSc3iTqQ/W/YX4QFCwoSsUJF4D) as `*.ndjson`
2. upload to saved objects page in Kibana
3. open ai assistant
4. use the connector selector to change between the new connectors and send a message to each one
5. confirm you get a descriptive error for the connectors labeled bad

<img width="1290" alt="Screenshot 2023-10-02 at 4 20 38 PM" src="https://github.com/elastic/kibana/assets/6935300/22e885f4-1002-4a40-a9e0-fe74367bc445">
